mythril/laser/plugin/plugins/plugin_annotations.py

```diff
@@ -18,15 +24,15 @@ def persist_over_calls(self) -> bool:
         return True
 
 
-class DependencyAnnotation(StateAnnotation):
+class DependencyAnnotation(MergeableStateAnnoation):
     """Dependency Annotation
 
     This annotation tracks read and write access to the state during each transaction.
     """
 
     def __init__(self):
         self.storage_loaded = []  # type: List
```

Comment:
Could this be a set as well?

Reply:
Yes, we can change it in a different PR, as I should also make changes in the dependency pruning file for this.

Reply:
Nevermind, making it a set seems easier when merging two annotations into a new variable.